data/parcels.json

```diff
@@ -2,6 +2,7 @@
     {
         "ref": "SD74445738",
         "totalPerimeter": 325.2,
+        "totalParcelArea": 1234.5,
```

Comment:
`data/parcels.json` in master now has `totalArea` for parcels, added as part of the new action ticket FCEP-90. The unit for these is hectares so we don't have to do any unit conversion from the quantity entered in the UI and rate defined in `data/actions.json.` which is £114 per ha.